jwst/assign_wcs/nirspec.py

```diff
@@ -1224,8 +1224,9 @@ def nrs_ifu_wcs(input_model):
                       'nrs_focus': imaging,
                       'nrs_mimf': imaging,
                       'nrs_bota': imaging,
-                      'nrs_autoflat': not_implemented_mode, #TBD
-                      'nrs_autowave': not_implemented_mode, #TBD
+                      'nrs_autoflat': not_implemented_mode,
+                      'nrs_autowave': not_implemented_mode,
                       'nrs_lamp': slits_wcs,
-                      'nrs_brightobj': slits_wcs
+                      'nrs_brightobj': slits_wcs,
+                      'nrs_dark': not_implemented_mode,
```

Comment:
I suppose, theoretically, INTFLAT could also be considered the same as imaging, as we're doing for SKYFLAT, but for several of the other instruments we've relegated all the flats to the not_implemented category. I think that most of the time the level-2b pipeline won't be run on flats and hence they won't be going through assign_wcs, but if someone were to ever try to run a flat through assign_wcs it could be useful to have it work (same as imaging).